src/multiples.rs

```diff
@@ -162,10 +162,15 @@ impl FromStr for Any {
     type Err = ParsingError;
 
     fn from_str(input: &str) -> Result<Any, Self::Err> {
-        match input {
+        // Special case this to mean kibibytes.
+        // FIXME(#10): this is not ideal.
+        if input == "KB" {
+            return Ok(Any::Kibibyte);
+        }
+        match &*input.to_uppercase() {
```

Comment:
This allocate a new string just for this match. Can you use something like `"b" | "B"` instead, that would avoid the allocation.

Reply:
I used `eq_ignore_ascii_case` :) to avoid having to quadruple the number of match patterns.